src/packages/excalidraw/README.md

```diff
@@ -132,7 +132,7 @@ export default function App() {
 | [`offsetLeft`](#offsetLeft)                   | Number                                                                                                                                                                                                                                                                                                                                             | `0`                  | left position relative to which Excalidraw should be rendered                                                                                              |
 | [`offsetTop`](#offsetTop)                     | Number                                                                                                                                                                                                                                                                                                                                             | `0`                  | top position relative to which Excalidraw should render                                                                                                    |
 | [`onChange`](#onChange)                       | Function                                                                                                                                                                                                                                                                                                                                           |                      | This callback is triggered whenever the component updates due to any change. This callback will receive the excalidraw elements and the current app state. |
-| [`initialData`](#initialData)                 | [`ExcalidrawElement[]`](https://github.com/excalidraw/excalidraw/blob/4c90ea5667d29effe8ec4a115e49efc7c340cdb3/src/types.ts#L151)                                                                                                                                                                                                                  | []                   | The initial data with which app loads.                                                                                                                     |
+| [`initialData`](#initialData)                 | <pre>{elements?: <a href="https://github.com/excalidraw/excalidraw/blob/4c90ea5667d29effe8ec4a115e49efc7c340cdb3/src/element/types.ts#L59">ExcalidrawElement[]</a>, appState?: <a href="https://github.com/excalidraw/excalidraw/blob/4c90ea5667d29effe8ec4a115e49efc7c340cdb3/src/types.ts#L33">AppState<a> } </pre>                              | null                 | The initial data with which app loads.                                                                                                                     |
```

Comment:
That would be more prone to breaking more frequently for example due to formatting/newline changes which I am trying to avoid and also when adding links in pr for new code it which is not merged to master it won't work  as mentioned https://github.com/excalidraw/excalidraw/pull/2644#discussion_r551046460 so we will need to come up with something better

Reply:
That's my point, I prefer to point to the latest master even if the line is wrong.. than to an outdated version. Devs will get that and most likely if they are already there will suggest a PR to fix it.. as it happens usually with dev docs..

Reply:
How do you do that for the new code which has been added to the PR?

Reply:
Make sure the PR is merged with latest master and use the line number from the PR. The url will stay the same (`/blob/master`)

Reply:
it should be `/blob/master` everywhere.

Reply:
Yea, I meant it as a reply to:

> How do you do that for the new code which has been added to the PR?